### ISSUE TITLE
Copying an address from tray icon menu does not work

### ISSUE THREAD

**Issue by SuperSweatyYeti** [f-koehler/KTailctl#36]
Trying to copy an address from the Ktailctl tray icon menu does not actually copy to the clipboard.

Copying an address from the full application window does work.

Using Fedora 38 KDE spin

**Comment by f-koehler** [comment]
I thought I already fixed this. But I just noticed something: copying from the tray icon works when the window is shown.
Then in the Qt docs I found this section: https://doc.qt.io/qt-5/qclipboard.html#notes-for-x11-users

It is about X11 but seems to be the case for wayland as well (which I am using). So I will have to figure out some custom code in the spirit of xclip/wlclipboard or maybe I can use something like https://github.com/dacap/clip
